travis/test_server.py

```diff
@@ -225,20 +305,22 @@ def main(argv=None):
               "Using '%s'" % odoo_version)
     test_loghandler = None
     if odoo_version == "6.1":
-        install_options += ["--test-disable"]
+        install_options += " --test-disable"
         test_loglevel = 'test'
     else:
-        options += ["--test-enable"]
+        test_options += " --test-enable"
         if odoo_version == '7.0':
             test_loglevel = 'test'
         else:
             test_loglevel = 'info'
             test_loghandler = 'openerp.tools.yaml_import:DEBUG'
     odoo_full = os.environ.get("ODOO_REPO", "odoo/odoo")
-    server_path = get_server_path(odoo_full, odoo_version, travis_home)
-    addons_path = get_addons_path(travis_dependencies_dir,
-                                  travis_build_dir,
-                                  server_path)
+    if not server_path:
+        server_path = get_server_path(odoo_full, odoo_version, travis_home)
```

Comment:
Do you dislike the compact form: ``server_path = server_path or get_server_path(...)`` ?